external/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala

```diff
@@ -68,6 +68,8 @@ abstract class AvroSuite
 
   override protected def beforeAll(): Unit = {
     super.beforeAll()
+    // initialize SessionCatalog here so it has a clean hadoopConf
+    spark.sessionState.catalog
```

Comment:
shall we do it in `SharedSparkSessionBase.initializeSession`?

Reply:
I thought about it but figured that might need a separate PR due to the impact?